src/sentry/api/endpoints/organization_member/index.py

```diff
@@ -29,27 +35,70 @@
 
 ERR_RATE_LIMITED = "You are being rate limited for too many invitations."
 
+# Required to explicitly define roles w/ descriptions because OrganizationMemberSerializer
+# has the wrong descriptions, includes deprecated admin, and excludes billing
+ROLE_CHOICES = [
+    ("billing", "Can manage payment and compliance details."),
```

Comment:
almost seems like this should be a map (dict) no?

Reply:
This is moved from @schew2381's code and I actually wasn't sure how docs read this vs. how serializer interpret this. So just asked. Let's see.

Reply:
for choicefield, the items are expected to be [tuples](https://docs.djangoproject.com/en/5.0/ref/models/fields/#field-choices)